src/helper.h

```diff
@@ -25,66 +25,7 @@ std::vector<char> binary_to_hex(char const *data, uint32_t len);
 
 std::vector<std::string> split(std::string const &input, std::string token);
 
-struct get_json_ret_string {
-  explicit operator bool() const;
-  explicit operator std::string() const;
-  bool found() const;
-  int err;
-  std::string v;
-};
-
-struct get_json_ret_int {
-  explicit operator bool() const;
-  explicit operator int() const;
-  explicit operator int64_t() const;
-  bool found() const;
-  int err;
-  int64_t v;
-};
-
-struct get_json_ret_uint {
-  explicit operator bool() const;
-  explicit operator uint64_t() const;
-  bool found() const;
-  int err;
-  uint64_t v;
-};
-
-struct get_json_ret_bool {
-  explicit operator bool() const;
-  bool found() const;
-  int v;
-};
-
-struct get_json_ret_array {
-  explicit operator bool() const;
-  bool found() const;
-  int err = 1;
-  rapidjson::Value const *v = nullptr;
-};
-
-struct get_json_ret_object {
-  explicit operator bool() const;
-  bool found() const;
-  int err = 1;
-  rapidjson::Value const *v = nullptr;
-};
-
-get_json_ret_string get_string(rapidjson::Value const *v, std::string path);
-get_json_ret_int get_int(rapidjson::Value const *v, std::string path);
-get_json_ret_uint get_uint(rapidjson::Value const *v, std::string path);
-get_json_ret_bool get_bool(rapidjson::Value const *v, std::string path);
-get_json_ret_array get_array(rapidjson::Value const &v, std::string path);
-get_json_ret_object get_object(rapidjson::Value const &v, std::string path);
-
-std::string pretty_print(rapidjson::Document const *v);
-
 template <typename T>
 std::pair<short int, T> to_num(const std::string &string) {
```

Comment:
I think `sleep_ms()` and all the `to_num()` overloads can also be removed. They don't seem to be used anywhere.